lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -111,6 +114,32 @@ def setUp(self):
         super(CourseTopicsViewTest, self).setUp()
         self.url = reverse("course_topics", kwargs={"course_id": unicode(self.course.id)})
 
+    def create_course(self, modules_count, module_store, topics):
+        """
+        Create a course in a specified module store with discussion module and topics
+        """
+        course = CourseFactory.create(
+            org="a",
+            course="b",
+            run="c",
```

Comment:
Nit: use better names, or just don't specify them and use the defaults provided by the `CourseFactory`.